files/manage_bootstrap_user.py

```diff
@@ -268,6 +277,7 @@ def delete_bootstrap_user(ini_file, username):
     args = parser.parse_args()
 
     if args.action == "create":
-        create_bootstrap_user(args.config, args.username, args.email, args.password, args.preset_api_key)
+        create_bootstrap_user(args.config, args.username, args.email,
```

Comment:
When breaking up long argument lines we usually make this one argument per line:
```suggestion
        create_bootstrap_user(args.config,
                                               args.username,
                                               args.email,
```